app.rb

```diff
@@ -0,0 +1,126 @@
+require_relative('./person')
+require_relative('./book')
+require_relative('./student')
+require_relative('./teacher')
+require_relative('./rental')
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def all_books
+    if @books.empty?
+      puts 'No books in the library yet'
+    else
+      @books.each do |book|
+        puts "Title: #{book.title} Author: #{book.author}"
+      end
+    end
+  end
+
+  def all_people
+    if @people.empty?
+      puts 'No people in the library yet'
+    else
+      @people.each do |person|
+        puts "class: #{person.class} ID: #{person.id} Name: #{person.name} Age: #{person.age}"
+      end
+    end
+  end
+
+  def create_person
+    print 'Do you want to create a Teacher (1) or teacher (2)? [input number]: '
```

Comment:
Well done on this project 👏🏾 

- [ ] I see a typo-mistake here in this `print 'Do you want to create a Teacher (1) or teacher (2)? [input number]: '`
You probably wanted to say `"Do you want to create a student (1) or a teacher (2)? [Input the number]: "`